middleware/logger.go

```diff
@@ -354,6 +365,24 @@ func logger(config LoggerConfig) fiber.Handler {
 	}
 }
 
+func nowTimeString(timeZone, layout string) string {
+	// This is different from Golang's time package which returns UTC, and Local is better than it
+	if timeZone == "" {
+		timeZone = "UTC"
+	}
+	location, err := time.LoadLocation(timeZone)
+	if err != nil {
+		log.Fatalf("Logger: failed to load time zone: %e\n", err)
```

Comment:
I think we should print the error and default to "UTC" instead of crashing the application.

Reply:
Because `middleware.LoggerConfig` does not use pointer, I did not find a way to change the `config` value from within the function. If I force the use of pointer, it may break the design of fibir.Could you give me some suggestions, please?

Reply:
That was indeed also my opinion. Crashing the application is not needed here imho

Reply:
> Because `middleware.LoggerConfig` does not use pointer, I did not find a way to change the `config` value from within the function. If I force the use of pointer, it may break the design of fibir.Could you give me some suggestions, please?

I think you can check TimeZone first. If it is invalid, set the value to `Local`

Reply:
Use the following logic after line 374
```go
log.Printf("logger: failed to load time zone: %e\n", err)
return time.Now().Format(layout)
```